schema/v5.0/CVE_JSON_5.0.schema

```diff
@@ -5,33 +5,45 @@
     "title": "",
     "description": "a CVE entry",
     "definitions": {
+        "uriType": {
+            "description": "A universal resource identifier (URI), according to [RFC 3986](https://tools.ietf.org/html/rfc3986).",
+            "type": "string",
+            "format": "uri",
+            "minLength": 1
+        },
+        "uuidType": {
+            "description": "A version 4 (random) universally unique identifier (UUID) as defined by [RFC 4122](https://tools.ietf.org/html/rfc4122#section-4.1.3).",
+            "type": "string",
+            "pattern": "^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-4[0-9A-Fa-f]{3}-[89ABab][0-9A-Fa-f]{3}-[0-9A-Fa-f]{12}$"
+        },
         "reference": {
             "type": "object",
-            "required": [
-                "url"
-            ],
+            "required": ["url"],
             "properties": {
                 "url": {
-                    "type": "string",
-                    "format":"uri",
-                    "description":"A universal resource identifier (URI), according to RFC3986."
+                    "description": "The uniform resource locator (URL), according to [RFC 3986](https://tools.ietf.org/html/rfc3986#section-1.1.3), that can be used to retrieve the referenced resource.",
+                    "$ref": "#/definitions/uriType"
                 },
                 "name": {
+                    "$comment": "TODO: description needed",
                     "type": "string",
                     "maxLength": 500,
                     "minLength": 1
                 },
                 "refSource": {
+                    "$comment": "QUESTION: is this still needed?",
                     "type": "string",
                     "maxLength": 500,
                     "minLength": 1,
                     "description": "name of the source, program, etc that produced the URL."
                 },
                 "tags": {
+                    "$comment": "TODO: description needed",
```

Comment:
"A list of one or more tags that describe the reference content (optional)."

Reply:
Maybe "an array of one or more tags that describe the reference resource (optional)."?